docs/source/reference/kubernetes/index.rst

```diff
@@ -34,6 +77,12 @@ To connect and use a Kubernetes cluster, SkyPilot needs:
 * An existing Kubernetes cluster running Kubernetes v1.20 or later.
 * A `Kubeconfig <https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/>`_ file containing access credentials and namespace to be used.
 
+**Supported Kubernetes deployments:**
+
+* Hosted Kubernetes services (EKS, GKE)
+* On-prem clusters (Kubeadm, Rancher, K3s)
+* Local development clusters (KinD, minikube)
```

Comment:
How about we rename `Submitting SkyPilot tasks to Kubernetes Clusters` to `Getting Started` and make it a subpage of `Run On Kubernetes` page?